routes/sectionRoutes.js

```diff
@@ -0,0 +1,30 @@
+const router = require('express').Router();
+const errorCodes = require('../helpers/errorCodes');
+const adminOnly = require('../middlewares/adminOnly');
+const mongoose = require('mongoose');
+
+// Models
+const { CourseModel } = require('../models/Courses');
+const { SectionModel } = require('../models/Sections');
+const { ComponentModel } = require('../models/Components');
+const { UserModel } = require('../models/Users');
+const {
+  ContentCreatorApplication,
+} = require('../models/ContentCreatorApplication');
+const requireLogin = require('../middlewares/requireLogin');
+const { IdentityStore } = require('aws-sdk');
```

Comment:
Could you perhaps clean up in the imports. From what I can see, only router and SectionModel is used.